static/js/views/discovered-addon.js

```diff
@@ -123,6 +129,55 @@ class DiscoveredAddon {
         controlDiv.innerHTML = el;
       });
   }
+  
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+    if(e.dataset.id) {
+      const license_url = "https://api.mozilla-iot.org:8443/addons/license/" + e.dataset.id;
+
+      var modal = document.getElementById('media-modal');
+
+      if(modal == null){
+        var modal_container = document.createElement('div');
```

Comment:
Use `let` or `const` instead of `var`. Also, use camelCase instead of snake_case for your variables.